Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -268,6 +279,12 @@ else if (useMegaTag2 == true)
       {
         setVisionMeasurementStdDevs(VecBuilder.fill(.7, .7, 9999999));
         addVisionMeasurement(mt2.pose, mt2.timestampSeconds);
+
+        fieldTypePub.set("Field2d");
```

Comment:
Using the fieldPub will add pose information to Network tables for X, Y, and rotation every 20 msec (or maybe 5 msec if it's called from Telemetry class). I'm OK with leaving this in for now, but if we're looking for NT traffic to reduce our disconnections, this is an immediate first choice due to its update frequency.

I also think this pose info is available from the LL network table as well. It's just not on our dashboard field display. But it CAN be in AdvantageScope.

Reply:
fixed